src/Symfony/Component/Yaml/Inline.php

```diff
@@ -59,7 +59,7 @@ public static function parse(string $value = null, int $flags = 0, array &$refer
     {
         self::initialize($flags);
 
-        $value = trim($value);
+        $value = null === $value ? '' : trim($value);
```

Comment:
The early return could be put first.

```suggestion
        if (null === $value) {
            return '';
        }

        self::initialize($flags);

        $value = trim($value);
```

We may change the return value to `null` is a future version.